### PR TITLE
Check Action: Get OOMKilled K8s pods

### PR DESCRIPTION
**Pull request by shloka-bhalgat-unskript**
### Testing
Screenshot 2023-06-08 at 7 22 27 PM

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

File: Kubernetes/legos/k8s_get_oomkilled_pods/k8s_get_oomkilled_pods.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    if handle.client_side_validation is not True:
+        raise ApiException(f"K8S Connector is invalid {handle}")
+
+    v1 = client.CoreV1Api(api_client=handle)
+    pods = v1.list_pod_for_all_namespaces().items
+    pods_to_check = pods
+    if namespace != "":
+        for p in pods:
+            if p.metadata.namespace == namespace:
+                pods_to_check = p
```
Comment (abhishek-unskript) [inline Kubernetes/legos/k8s_get_oomkilled_pods/k8s_get_oomkilled_pods.py:48]: This should have been append()

Comment (abhishek-unskript) [reply Kubernetes/legos/k8s_get_oomkilled_pods/k8s_get_oomkilled_pods.py:48]: AI suggested: 

// given a kubernetes namespace, list all pods in that namespace, if no valid namespace is given, list all pods. write this in python

```
from kubernetes import client, config

def list_oomkilled_pods_in_namespace(namespace=None):
    # Loads kubeconfig from default location (~/.kube/config)
    # or KUBECONFIG environment variable.
    config.load_kube_config()

    v1 = client.CoreV1Api()
    
    if namespace:
        try:
            print(f"Listing OOMKilled pods with namespace {namespace}:")
            pods = v1.list_namespaced_pod(namespace)
        except client.rest.ApiException as e:
            print(f"Exception when calling CoreV1Api->list_namespaced_pod: {e}\nInvalid namespace")
            return
    else:
        print("Listing all OOMKilled pods:")
        pods = v1.list_pod_for_all_namespaces()

    for pod in pods.items:
        for status in pod.status.container_statuses:
            if status.state.terminated and status.state.terminated.reason == 'OOMKilled':
                print(f"Pod: {pod.metadata.name}, Namespace: {pod.metadata.namespace}, Container: {status.name}")

# Replace 'my-namespace' with your namespace
# To list all pods, just call the function with no arguments
list_oomkilled_pods_in_namespace('my-namespace')
```

**Review by abhishek-unskript — CHANGES_REQUESTED** [review]
Please see comments..

File: Kubernetes/legos/k8s_get_oomkilled_pods/k8s_get_oomkilled_pods.py (hunk 1 above)
Comment (shloka-bhalgat-unskript) [reply Kubernetes/legos/k8s_get_oomkilled_pods/k8s_get_oomkilled_pods.py]: Changed the code to get all namespaces.

**Review by abhishek-unskript — APPROVED** [review]

**Pull request merged**